packages/documentation/src/stories/components/accordion/accordion.stories.ts

```diff
@@ -80,7 +81,7 @@ export const MultipleOpenPanels: Story = {
 
 export const DefaultCollapsedPanels: Story = {
   render: () => html`
-    <post-accordion>
+    <post-accordion heading-level="4">
```

Comment:
Why did you not use the storybook logic to override args on the story level here?